### PR TITLE
Various NOUN, PROPN, and VERB feature fixes

### PR DESCRIPTION
**Pull request by rhdunn**
This adds various rules to the neatEN Python script to check for issues identified in #360, along with some others identified while doing this. These check:
1. That subsets of UPenn XPOS map to corresponding UPOS tags.
2. That specific features correspond to XPOS tags.

The identified errors have been corrected.

Note that this is not a complete list of feature checks.

### CODE REVIEW

**Comment by nschneid** [comment]
Excellent. I will get to this once some other updates are merged.

**Comment by rhdunn** [comment]
I've fixed a couple of issues, and extended this to more cases. I've not done a full set as some XPOS tags (RB, VBP, etc.) are missing features, so generate a lot of warnings/output. As such, I'm not intending on adding any additional rules/checks in this PR.

**Comment by nschneid** [comment]
I have merged the other updates. There are a few conflicts.

File: en_ewt-ud-dev.conllu [hunk 1]
```diff
@@ -24644,7 +24644,7 @@
 7	fish	fish	NOUN	NN	Number=Sing	0	root	0:root	_
 8	-	-	PUNCT	,	_	7	punct	7:punct	_
 9	but	but	CCONJ	CC	_	11	cc	11:cc	_
-10	u	you	PRON	PRP	Abbr=Yes|Person=2|Poss=Yes|PronType=Prs	11	nsubj	11:nsubj|13:nsubj:xsubj|18:nsubj:xsubj	_
+10	u	you	PRON	PRP$	Abbr=Yes|Person=2|Poss=Yes|PronType=Prs	11	nsubj	11:nsubj|13:nsubj:xsubj|18:nsubj:xsubj	_
```
Comment (nschneid) [inline en_ewt-ud-dev.conllu:24647]: PRP is correct

Comment (nschneid) [reply en_ewt-ud-dev.conllu:24647]: (should not be `Poss=Yes`)

File: en_ewt-ud-test.conllu [hunk 2]
```diff
@@ -23443,7 +23443,7 @@
 
 # sent_id = answers-20111106103415AAqdokn_ans-0005
 # text = Fudruckers.
-1	Fudruckers	Fuddruckers	PROPN	NNPS	Number=Sing|Typo=Yes	0	root	0:root	CorrectForm=Fuddruckers|SpaceAfter=No
+1	Fudruckers	Fuddrucker	PROPN	NNPS	Number=Plur|Typo=Yes	0	root	0:root	CorrectForm=Fuddruckers|SpaceAfter=No
```
Comment (nschneid) [inline en_ewt-ud-test.conllu:23446]: Should be `NNP` (company name ends in "s")

File: not-to-release/sources/answers/20111108105749AABv7vx_ans.xml.conllu [hunk 3]
```diff
@@ -546,7 +546,7 @@
 12	the	the	DET	DT	Definite=Def|PronType=Art	13	det	13:det	_
 13	room	room	NOUN	NN	Number=Sing	15	nsubj	15:nsubj	_
 14	is	be	AUX	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	15	cop	15:cop	_
-15	his	his	PRON	PRP	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	10	ccomp	10:ccomp	_
+15	his	his	PRON	PRP$	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	10	ccomp	10:ccomp	_
```
Comment (nschneid) [inline not-to-release/sources/answers/20111108105749AABv7vx_ans.xml.conllu:549]: `PRP` is correct. This is the independent form of the pronoun (cf. "theirs"). Per just-decided guidelines (UniversalDependencies/docs#517), the features on this token are correct (`Case=Gen` applies to dependent ones only).

**Comment by nschneid** [comment]
Also note the new policy on possessive lemmas & features: https://github.com/UniversalDependencies/docs/issues/517#issuecomment-1272400316 Do you want to implement that or do you want me to? Either way, it does not affect XPOS.

File: en_ewt-ud-test.conllu (hunk 2 above)
Comment (rhdunn) [reply en_ewt-ud-test.conllu]: Thanks. I've fixed this during the merge.

**Comment by rhdunn** [comment]
With the merge, I've adopted your changes from the clefts branch when resolving the conflicts.

File: en_ewt-ud-dev.conllu (hunk 1 above)
Comment (rhdunn) [reply en_ewt-ud-dev.conllu:24647]: Fixed. Thanks.

File: en_ewt-ud-test.conllu (hunk 2 above)
Comment (rhdunn) [reply en_ewt-ud-test.conllu]: Fixed. Thanks.

**Comment by rhdunn** [comment]
I've fixed the other issues you mentioned now.

File: not-to-release/sources/answers/20111108105749AABv7vx_ans.xml.conllu (hunk 3 above)
Comment (rhdunn) [reply not-to-release/sources/answers/20111108105749AABv7vx_ans.xml.conllu]: Fixed. Thanks.

**Comment by rhdunn** [comment]
I'll have a look at applying the changes from [docs#517](https://github.com/UniversalDependencies/docs/issues/517). I think it makes sense to do that as a separate PR/branch.

File: en_ewt-ud-train.conllu [hunk 4]
```diff
@@ -92611,7 +92611,7 @@
 2	a	a	DET	DT	Definite=Ind|PronType=Art	3	det	3:det	_
 3	location	location	NOUN	NN	Number=Sing	13	nsubj:pass	13:nsubj:pass	SpaceAfter=No
 4	,	,	PUNCT	,	_	3	punct	3:punct	_
-5	Theirs	theirs	PRON	PRP	Number=Plur|Person=3|Poss=Yes|PronType=Prs	3	appos	3:appos	_
+5	Theirs	theirs	PRON	PRP$	Number=Plur|Person=3|Poss=Yes|PronType=Prs	3	appos	3:appos	_
```
Comment (nschneid) [inline en_ewt-ud-train.conllu:92614]: PRP is correct (independent form)

File: en_ewt-ud-train.conllu [hunk 5]
```diff
@@ -174898,15 +174898,15 @@
 14	and	and	CCONJ	CC	_	15	cc	15:cc	_
 15	toys	toy	NOUN	NNS	Number=Plur	13	conj	13:conj:and|17:nsubj	_
 16	are	be	AUX	VBP	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	17	cop	17:cop	_
-17	his	his	PRON	PRP	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	7	conj	5:obj|7:conj:and	SpaceAfter=No
+17	his	his	PRON	PRP$	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	7	conj	5:obj|7:conj:and	SpaceAfter=No
```
Comment (nschneid) [inline en_ewt-ud-train.conllu:174901]: PRP

File: en_ewt-ud-train.conllu [hunk 6]
```diff
@@ -175069,7 +175069,7 @@
 9	have	have	AUX	VB	VerbForm=Inf	10	aux	10:aux	_
 10	made	make	VERB	VBN	Tense=Past|VerbForm=Part	0	root	0:root	_
 11	it	it	PRON	PRP	Case=Acc|Gender=Neut|Number=Sing|Person=3|PronType=Prs	10	obj	10:obj|12:nsubj:xsubj	_
-12	his	his	PRON	PRP	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	10	xcomp	10:xcomp	SpaceAfter=No
+12	his	his	PRON	PRP$	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	10	xcomp	10:xcomp	SpaceAfter=No
```
Comment (nschneid) [inline en_ewt-ud-train.conllu:175072]: PRP

File: en_ewt-ud-train.conllu [hunk 7]
```diff
@@ -175168,7 +175168,7 @@
 5	the	the	DET	DT	Definite=Def|PronType=Art	7	det	7:det	_
 6	whole	whole	ADJ	JJ	Degree=Pos	7	amod	7:amod	_
 7	house	house	NOUN	NN	Number=Sing	4	obj	4:obj|8:nsubj:xsubj	_
-8	his	his	PRON	PRP	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	4	xcomp	4:xcomp	SpaceAfter=No
+8	his	his	PRON	PRP$	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	4	xcomp	4:xcomp	SpaceAfter=No
```
Comment (nschneid) [inline en_ewt-ud-train.conllu:175171]: PRP

File: en_ewt-ud-train.conllu [hunk 8]
```diff
@@ -181477,7 +181477,7 @@
 18	the	the	DET	DT	Definite=Def|PronType=Art	19	det	19:det	_
 19	house	house	NOUN	NN	Number=Sing	16	nmod	16:nmod:of	_
 20	is	be	AUX	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	21	cop	21:cop	_
-21	his	his	PRON	PRP	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	14	ccomp	14:ccomp	SpaceAfter=No
+21	his	his	PRON	PRP$	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	14	ccomp	14:ccomp	SpaceAfter=No
```
Comment (nschneid) [inline en_ewt-ud-train.conllu:181480]: PRP

File: en_ewt-ud-train.conllu [hunk 9]
```diff
@@ -197015,7 +197015,7 @@
 # sent_id = reviews-348369-0006
 # text = Tmobile want to Send of my phone and i didn't want to go thru that...
 1	Tmobile	Tmobile	PROPN	NNP	Number=Sing	2	nsubj	2:nsubj|4:nsubj:xsubj	_
-2	want	want	VERB	VBZ	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	0:root	_
+2	want	want	VERB	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	0:root	_
```
Comment (nschneid) [inline en_ewt-ud-train.conllu:197018]: Isn't this the British collective plural? VBP?

File: en_ewt-ud-train.conllu [hunk 10]
```diff
@@ -246794,7 +246794,7 @@
 27	that	that	PRON	WDT	PronType=Rel	29	obj	26:ref	_
 28	I	I	PRON	PRP	Case=Nom|Number=Sing|Person=1|PronType=Prs	29	nsubj	29:nsubj	_
 29	loved	love	VERB	VBD	Mood=Ind|Number=Sing|Person=1|Tense=Past|VerbForm=Fin	26	acl:relcl	26:acl:relcl	_
-30	compared	compare	VERB	VBN	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	20	obl	20:obl:about	_
+30	compared	compare	VERB	VBD	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	20	obl	20:obl:about	_
```
Comment (nschneid) [inline en_ewt-ud-train.conllu:246797]: I think VBN is correct. Should be VerbForm=Part

**Comment by nschneid** [comment]
@rhdunn Now that personal pronouns are updated, is it worth finishing this?

**Comment by rhdunn** [comment]
That's my intention, yes.

File: en_ewt-ud-train.conllu (hunk 10 above)
Comment (rhdunn) [reply en_ewt-ud-train.conllu:246797]: Fixed, thanks.

File: en_ewt-ud-train.conllu (hunk 9 above)
Comment (rhdunn) [reply en_ewt-ud-train.conllu:197018]: Fixed, thanks.

File: en_ewt-ud-train.conllu (hunk 8 above)
Comment (rhdunn) [reply en_ewt-ud-train.conllu]: Fixed, thanks.

File: en_ewt-ud-train.conllu (hunk 7 above)
Comment (rhdunn) [reply en_ewt-ud-train.conllu]: Fixed, thanks.

File: en_ewt-ud-train.conllu (hunk 6 above)
Comment (rhdunn) [reply en_ewt-ud-train.conllu]: Fixed, thanks.

File: en_ewt-ud-train.conllu (hunk 5 above)
Comment (rhdunn) [reply en_ewt-ud-train.conllu]: Fixed, thanks.

File: en_ewt-ud-train.conllu (hunk 4 above)
Comment (rhdunn) [reply en_ewt-ud-train.conllu]: Fixed, thanks.

**Comment by rhdunn** [comment]
This is ready to merge now if you are happy with the fixes.

File: en_ewt-ud-train.conllu (hunk 10 above)
Comment (nschneid) [reply en_ewt-ud-train.conllu:246797]: wait never mind

**Pull request merged**